dts/bindings/sensor/ti,ina219.yaml

```diff
@@ -0,0 +1,107 @@
+# Copyright (c) 2021 Leonard Pollak
+# SPDX-License-Identifier: Apache-2.0
+
+description: Texas Instruments Bidirectional Current/Power Sensor
+
+compatible: "ti,ina219"
+
+include: i2c-device.yaml
+
+properties:
+    lsb-microamp:
+      type: int
+      required: false
+      default: 100
```

Comment:
Justifications for all defaults must be provided; please see https://docs.zephyrproject.org/latest/guides/dts/bindings.html#default-values-for-properties for the rules.

Reply:
Good catch! It doesn't seem right to have a default value for this. I will change it accordingly.

Reply:
OK, thanks! Please note that the requirements in that link apply to all defaults in this PR

Reply:
I pushed the change. All other defaults correspond to the default power-up state of the config register.

Reply:
> All other defaults correspond to the default power-up state of the config register.

Thanks for confirming that, but you need to explicitly document that in the binding. From the bindings link I posted (emphasis mine):

"In any case where default: is used, the property documentation should **explain why the value was selected** and any conditions that would make it necessary to provide a different value. (**This is mandatory for built-in bindings.**)"

Reply:
Sorry for the inconvenience but I didn't read up to that paragraph.
I have now added (what I think is) all the necessary documentation.